packages/scandipwa/src/component/Popup/Popup.style.scss

```diff
@@ -30,7 +30,7 @@
 .Popup {
     opacity: 0;
     pointer-events: none;
-    z-index: 400;
+    z-index: 999;
```

Comment:
Tho, isn't there value that goes more then 400? like cookie popup. Should it appear on top of all popups?

Reply:
Hey, so HeaderPopup had 400, I applied 450 over image and form popup. And cookies and new version popups with 460 and 455 respectively.